sdks/python/scripts/run_integration_test.sh

```diff
@@ -171,12 +171,11 @@ fi
 
 if [[ -z $PIPELINE_OPTS ]]; then
 
-  # Create a tarball if not exists
+  # Get tar ball path
   if [[ $(find ${SDK_LOCATION} 2> /dev/null) ]]; then
-    SDK_LOCATION=$(find ${SDK_LOCATION})
+    SDK_LOCATION=$(find ${SDK_LOCATION} | tail -n1)
   else
-    python setup.py -q sdist
-    SDK_LOCATION=$(ls dist/apache-beam-*.tar.gz | tail -n1)
+    echo '[WARNING] Did not find tar ball from SDK_LOCATION.'
```

Comment:
What will happen when there is no tarball? Will the suite fail?

Reply:
yes, pipeline will fail in staging phase before job submission.

Reply:
How about this warning:
`[WARNING] Could not find SDK tarball in $SDK_LOCATION.`